ctms/bin/acoustic_sync.py

```diff
@@ -1,13 +1,46 @@
 #!/usr/bin/env python3
 """Run continuously in the background, syncing acoustic with our db."""
+import argparse
+import logging
 from time import monotonic, sleep
 
 from ctms import config
 from ctms.database import get_db_engine
 from ctms.sync import CTMSToAcousticSync
 
+LOGGER = None
+
+
+def _setup_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-log",
+        "--log",
+        default="debug",
+        help=("provide logging level to job (default: debug)"),
+    )
+
+    options = parser.parse_args()
+    levels = {
+        "critical": logging.CRITICAL,
+        "error": logging.ERROR,
+        "warn": logging.WARNING,
+        "warning": logging.WARNING,
+        "info": logging.INFO,
+        "debug": logging.DEBUG,
+    }
+    level = levels.get(options.log.lower())
+    if level is None:
+        raise ValueError(
+            f"log level given: {options.log}"
+            f" -- must be one of: {' | '.join(levels.keys())}"
+        )
+    logging.basicConfig(level=level)
```

Comment:
I was hoping that ``ctms.log.configure_logging`` would be shared by the background jobs. We can look into merging later, though.

Reply:
Opted to using this helper. Thanks for pointing it out.